ape_hardhat/providers.py

```diff
@@ -367,10 +391,12 @@ def connect(self):
         upstream_genesis_block_hash = self._upstream_provider.get_block(0).hash
         self._upstream_provider.disconnect()
         if self.get_block(0).hash != upstream_genesis_block_hash:
-            self.disconnect()
-            raise HardhatProviderError(
-                f"Upstream network is not {self.network.name.replace('-fork', '')}"
+            # self.disconnect()
```

Comment:
Should we just remove these commented out bits?

Reply:
I figured to keep them in for later when the change reverts. I can mention this in a comment though

Reply:
Ah, maybe a TODO or something?